Source/JavaScriptCore/interpreter/Interpreter.cpp

```diff
@@ -728,12 +729,22 @@ class UnwindFunctor {
 #if ENABLE(WEBASSEMBLY)
                 if (m_catchableFromWasm) {
                     auto* wasmCallee = static_cast<Wasm::Callee*>(nativeCallee);
+                    if (wasmCallee->compilationMode() == Wasm::CompilationMode::WasmToJSMode) {
+                        // https://webassembly.github.io/exception-handling/js-api/#create-a-host-function
+                        if (!m_wasmTag)
+                            m_wasmTag = &Wasm::Tag::jsExceptionTag();
+                    }
```

Comment:
It's probably not super perf important but why not hoist this to the constructor? That would take it out of the loop.

Reply:
This is important since we would like to set this tag only when we see WasmToJSMode.